### ISSUE TITLE
Please update the following components: LoadableComponent

### ISSUE THREAD

**Issue by pshevtsov** [prescottprue/generator-react-firebase#186]
**Describe the bug**

I get warning messages: “Warning: `componentWillMount` has been renamed” in my DevTools console.

**To Reproduce**

1. Generate the app and start the dev server: `yo react-firebase && npm start`
1. Sign-up and login
1. Go to http://localhost:3000/projects
1. Open Browser's DevTools console and see error:

```
Warning: componentWillMount has been renamed, and is not recommended for use. See https://fb.me/react-unsafe-component-lifecycles for details.

* Move code with side effects to componentDidMount, and set initial state in the constructor.
* Rename componentWillMount to UNSAFE_componentWillMount to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: LoadableComponent
```

**Expected behavior/code**
No warning message.

**Comment by prescottprue** [comment]
Thanks for noting this, I will look into it. I believe it may be due to `react-loadable`

**Comment by prescottprue** [comment]
Thanks for reporting! Fixed in [v5.2.0 release](https://github.com/prescottprue/generator-react-firebase/releases/tag/v5.2.0) by switching to `React.lazy` and `React.Suspense`
